docs/documentation/github-desktop.md

```diff
@@ -0,0 +1,38 @@
+# Store your code online with GitHub and GitHub Desktop
+
+GitHub is a way to store code online so you can collaborate with other people. It also makes it easier to publish your prototype online with Heroku.
+
+Some concepts:
+
+- Git - a way to store versions of your code
```

Comment:
Does the word 'way' sound like we're talking about a method, as opposed to a _thing_? Maybe we could say "a web-based system for tracking file versions" (or similar).

Reply:
agree 'way' sounds like a method, the problem is a better word - system also sounds a bit like method :)

git itself isnt web based, thats github

I think file versions isnt quite right - I think its an important distinction that it track your whole project - a commit represents the state of a whole project at once

Reply:
What about: "software that stores versions of your code"?